graph/topo/bench_test.go

```diff
@@ -56,7 +70,6 @@ func benchmarkTarjanSCC(b *testing.B, g graph.Directed) {
 		b.Fatal("unexpected number zero-sized SCC set")
 	}
 }
-
```

Comment:
Please revert these line deletions.

Reply:
Oops, okay, done!

Any particular reason why you wanted these deletions reverted but not have blank lines between `benchmarkSort`/`BenchmarkSort*` and `benchmarkSortStabilized`/`BenchmarkSortStabilized*`, or was that just a oversight?

Reply:
Please fix those too.

Reply:
Okay, done!